framework/sandstone.h

```diff
@@ -82,6 +83,21 @@ extern "C" {
 #  define log_debug( ...)       (void)0
 #endif
 
+// skip categories
+typedef enum SkipCategory{
+    RESOURCE_UNAVAILABLE=65,
+    CPU_NOT_SUPPORTED=66,
+    DEVICE_NOT_FOUND=67,
+    DEVICE_NOT_CONFIGURED=68,
+    OTHERS=69,
```

Comment:
What `OTHERS` is supposed to mean? First of all, it should be singular. Secondly, is `UNKNOWN` is a better name for this?

Reply:
Will change accordingly. Changing this too which will have singular values.